### ISSUE TITLE
merge picture&foaf:description, pictureDescription&depictionDescription

### ISSUE THREAD

**Issue by VladimirAlexiev** [dbpedia/mappings-tracker#85]
The following prop pairs mean the same, so merge them:
- picture&foaf:description
- pictureDescription&depictionDescription

I think "picture" is used more often, but needs to be checked.

In addition, check the usage of imageSize. It could just be deleted, since all (most?) images are now on Commons, and commons metadata is emitted standardly for all media (no mapping needed).

**Comment by jimkont** [comment]
mapping images actually doesn't work atm and we use the ImageExtractor for this.
so both dbo:picture and foaf:depiction doesn't do anything

however, iirc foaf:depiction is used by the framework for the image extractor and cannot be deleted while dbo:picture can

**Comment by VladimirAlexiev** [comment]
I agree: one is objectProperty the other is dataProperty, but neither makes an image URL correctly.
- Does the ImageExtractor also extract caption and imageSize?
- How about different kinds of images: image, signature, coatOfArms, map... ?
